docs/source/reference/types.rst

```diff
@@ -76,7 +76,7 @@ complex128              c16              double-precision complex number
 Arrays
 ------
 
-The easy way to declare array types is to subscript an elementary type
+The easy way to declare :class:`~numba.types.Array` types is to subscript an elementary type
```

Comment:
Please line wrap to 80 characters, thanks.

Reply:
It looks like https://github.com/numba/numba/pull/6843/commits/8452bdefa986fa4113f53196b32cf53764319e01 changed the line below and not this line?

Reply:
Yes, I am sorry. I will correct this.